integrations/docker/images/stage-2/chip-build-efr32/Dockerfile

```diff
@@ -33,6 +33,14 @@ RUN wget https://www.silabs.com/documents/login/software/slc_cli_linux.zip && \
 
 ENV PATH="${PATH}:/slc_cli/"
 
+#Clone Wiseconnect SDK 3.1.0-MatterCert (f84bf66)
+RUN git clone --depth=1 --branch=MatterCert https://github.com/SiliconLabs/wiseconnect.git wifi_sdk && \
+    cd wifi_sdk && \
+    rm -rf .git \
+    && : # last line
+
+ENV WISECONNECT_PATH=/wifi_sdk/
```

Comment:
we probably should update and set paths in the vscode image as well.

In that image we probably should place wifi_sdk in some more silabs/efr path since it must co-exist with other SDKs as well.

Reply:
There is no place where the efr32 path is mentioned in the vscode image i.e., in integrations/docker/images/vscode/chip-build-vscode/Dockerfile. Should the environment path be added there with `ENV WISECONNECT_PATH=/wifi_sdk/` ?

Reply:
The intent is that vscode image should be usable for any build, the same like `chip-build-<PLATFORM>` is for all platforms.

In particular here, it seems WISECONNECT_PATH may be used by the build system, so I would expect vscode image to:
  - copy this SDK into the image
  - set WISECONNECT_PATH